lib/ansible/modules/files/file.py

```diff
@@ -296,7 +296,7 @@ def get_state(path):
 
     b_path = to_bytes(path, errors='surrogate_or_strict')
     try:
-        if os.path.lexists(b_path):
+        if os.lstat(b_path):
```

Comment:
This is not quite the correct fix. We probably need to use something `os.access()` to check if the file exists and if we can access it:
```python
if os.access(b_path, os.F_OK):  # checks that the file exists, following symlinks
    if not os.access(b_path, os.R_OK):  # check that we can read the file
        # probably warn, for now, then change to an error later